spec/integration/data_stream/manifest.spec.yml

```diff
@@ -102,6 +104,39 @@ spec:
           default:
             description: Default value(s) for variable
             $ref: "#/definitions/input_variable_value"
+        if:
```

Comment:
Oh nice, maybe we can use if/then in more cases where we are validating with go now.